audio/midi/fluid/sfont.cpp

```diff
@@ -1645,16 +1673,19 @@ void SFont::load_shdr (int size)
                   continue;
                   }
             if ((p->end > getSamplesize()) || (p->start > (p->end - 4))) {
-                  qWarning("Sample start/end file positions are invalid, disabling");
+                  qWarning("Sample(%s) start/end file positions are invalid, disabling", p->name);
```

Comment:
shouldn't there be a space before the `(%s)`?

Reply:
I find this easier to read (the string directly relates to Sample), and the alternatives would be…

    qWarning("Sample %s start/end…

… which is bad for strings with arbitrary contents, and…

    qWarning("Sample \"%s\" start/end…

… which is just uglier, and the way I used has something more of a programmer style.

Sorry, I have a headache and can barely think ,let alone formulate right.

Reply:
Probably not woth loosing much sweat on, as most users would never see those anyhow